src/pages/registration/files_and_license_tab/FileUploader.tsx

```diff
@@ -14,9 +14,11 @@ type UploadedFile = Pick<
   'type' | 'identifier' | 'name' | 'size' | 'mimeType' | 'rightsRetentionStrategy' | 'uploadDetails'
 >;
 
+let hasUploadSuccessEventListener = false;
+
 export const FileUploader = ({ addFile, uppy }: FileUploaderProps) => {
   useEffect(() => {
-    if (uppy && !uppy.opts.meta.hasUploadSuccessEventListener) {
+    if (uppy && !hasUploadSuccessEventListener) {
       uppy.on('upload-success', (file, response) => {
```

Comment:
kan useUppyEvent være en bedre løsning for dette?
https://uppy.io/docs/react/#useuppyeventuppy-event-callback